circle.yml

```diff
@@ -1,4 +1,17 @@
 machine:
   node:
     version: 4.3.0
-
+test:
+  override:
+    - ./node_modules/.bin/eslint --quiet:
```

Comment:
Do we worry about the maintainability of splitting this off of the `lint` Makefile target?

(First time I've noticed `mixedindentlint` as a dependent target of `lint`)